girder/cli/serve.py

```diff
@@ -5,17 +5,28 @@
 
 from girder import _attachFileLogHandlers
 from girder.utility import server
+from girder.constants import ServerMode
 
 
 @click.command(name='serve', short_help='Run the Girder server.', help='Run the Girder server.')
-@click.option('-t', '--testing', is_flag=True, help='Run in testing mode')
+@click.option('--dev', default=False, is_flag=True, help='Sets the server mode to development')
+@click.option('--mode', type=click.Choice([
+    ServerMode.PRODUCTION,
+    ServerMode.DEVELOPMENT,
+    ServerMode.TESTING
```

Comment:
What runs `girder serve --mode=testing`?

Reply:
I'm not sure, I was just preserving what was already there.